src/test/kotlin/io/bazel/kotlin/builder/KotlinAbstractTestBuilder.java

```diff
@@ -124,35 +126,57 @@ public final Path writeGeneratedSourceFile(String filename, String[] lines) {
     return writeFile(DirectoryType.SOURCE_GEN, filename, lines);
   }
 
+  private static void assertFileExistence(Stream<Path> pathStream, boolean shouldExist) {
```

Comment:
This method is confusing. I don't see an actual existence test, just behavior based on the should boolean.

Reply:
path.toFile().exists() is a test, albeit the File api, not the Path api.

It was like this when I got here -- Since it's private, it can wait until the test structure is refactored.